### PR TITLE
openjdk: updates

### PR DESCRIPTION
**Pull request by xtexx**
Topic Description
-----------------

- openjfx-22: new, 22.0.2+ga
- openjdk-22: new, 22.0.2+ga
- openjfx: update to 21.0.5+0
- openjdk: update to 21.0.4-ga
- openjfx-17: new, 17.0.13+0
    - new, 17.0.9+3 (derive from openjfx)
    - update to 17.0.13+0
- openjdk-17: new, 17.0.12+ga
- openjdk-8: update to 8u422+ga
    - update to 8u422+ga
    - add loongson3 patch
    - add stage2 build script
    - remove webkit from amd64 due to build failures
- openjfx: use openjfx-common
- openjfx-11: use openjfx-common
- openjfx-8: use openjfx-common

... and 3 more commits

Package(s) Affected
-------------------

- openjdk: 3:21.0.4+ga
- openjdk-17: 3:17.0.12+ga
- openjdk-22: 3:22.0.2+ga
- openjdk-8: 3:8u422+ga
- openjfx: 21.0.5+0
- openjfx-11: 11.0.20+1-1
- openjfx-17: 17.0.13+0
- openjfx-22: 22.0.2+ga
- openjfx-8: 8u202+ga-12
- openjfx-common: 1

Security Update?
----------------

No

Build Order
-----------

```
#buildit openjfx-common openjdk-8:+stage2 openjdk-8 openjfx-8 openjfx-11 openjdk-17:+stage2 openjdk-17 openjfx-17 openjdk:+stage2 openjdk openjfx openjfx-22 openjdk-22:+stage2 openjdk-22
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by xtexx** [comment]
```
# riscv64 is not supported by openjdk 8

# TESTING BUILD
/build openjdk-22-new openjdk-8:+stage2,openjdk-8,openjfx-common,openjfx-8 amd64,arm64,ppc64el,loongson3,loongarch64
/build openjdk-22-new openjfx-common,openjfx-11 amd64,arm64,ppc64el,riscv64
/build openjdk-22-new openjdk-17:+stage2,openjdk-17,openjfx-common,openjfx-17 amd64,arm64,ppc64el,loongson3,loongarch64,riscv64
/build openjdk-22-new openjdk:+stage2,openjdk,openjfx-common,openjfx,openjfx-22 amd64,arm64,ppc64el,loongson3,loongarch64,riscv64
/build openjdk-22-new openjdk:+stage2,openjdk-22:+stage2,openjdk-22 amd64,arm64,ppc64el,loongson3,loongarch64,riscv64

# STABLE BUILD
/build stable openjfx-common amd64
# wait for openjfx-common, then parallel run
/build stable openjdk-8:+stage2,openjdk-8,openjfx-8 amd64,arm64,ppc64el,loongson3,loongarch64
/build stable openjfx-11 amd64,arm64,ppc64el,riscv64
/build stable openjdk-17:+stage2,openjdk-17,openjfx-17 amd64,arm64,ppc64el,loongson3,loongarch64,riscv64
/build stable openjdk:+stage2,openjdk,openjfx,openjdk-22:+stage2,openjdk-22,openjfx-22 amd64,arm64,ppc64el,loongson3,loongarch64,riscv64

# OPENPR
/openpr openjdk,openjfx: updates;openjdk-22-new;openjfx-common,openjdk-8:+stage2,openjdk-8,openjfx-8,openjfx-11,openjdk-17:+stage2,openjdk-17,openjfx-17,openjdk:+stage2,openjdk,openjfx,openjfx-22,openjdk-22:+stage2,openjdk-22;new-packages,upgrade,stage2
```

**Comment by xtexx** [comment]
- https://buildit.aosc.io/pipelines/9117 (JDK 8, JFX 8)
- https://buildit.aosc.io/pipelines/9118 (JFX 11)
- https://buildit.aosc.io/pipelines/9122 (JDK 17, JFX 17)
- https://buildit.aosc.io/pipelines/9120 (JDK 21, JFX 21, JFX 22)
- https://buildit.aosc.io/pipelines/9121 (JDK 21, JDK 22)

File: lang-java/openjfx-common/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PKGNAME=openjfx-common
+PKGSEC=java
+PKGDES="Common files for OpenJFX"
+
+ABHOST="noarch"
```
Comment (MingcongBai) [inline lang-java/openjfx-common/autobuild/defines:5]: Missing PKGBREAK + REP for openjfx (version in current stable).

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]
Tested very good on LoongArch (Minecraft via HMCL, with JDK/JFX 17/21/22):

Screenshot_20240812_151626
Screenshot_20240812_152128
Screenshot_20240812_151828

Java 8 also runs, which I can only assume works well since it built itself.

**Pull request merged**